src/main/kotlin/com/ampnet/blockchainapiservice/service/SendErc20RequestService.kt

```diff
@@ -1,11 +1,14 @@
 package com.ampnet.blockchainapiservice.service
 
 import com.ampnet.blockchainapiservice.model.params.CreateSendErc20RequestParams
+import com.ampnet.blockchainapiservice.model.result.FullSendErc20Request
 import com.ampnet.blockchainapiservice.model.result.SendErc20Request
+import com.ampnet.blockchainapiservice.util.TransactionHash
 import com.ampnet.blockchainapiservice.util.WithFunctionData
 import java.util.UUID
 
 interface SendErc20RequestService {
     fun createSendErc20Request(params: CreateSendErc20RequestParams): WithFunctionData<SendErc20Request>
-    fun attachTxHash(id: UUID, txHash: String)
+    fun getSendErc20Request(id: UUID, rpcUrl: String?): FullSendErc20Request
+    fun attachTxHash(id: UUID, txHash: TransactionHash)
```

Comment:
This is preparation for the next PR? It's not used?

Reply:
Yes, it will be used in controllers.